packages/zarathustra/protocols/asset_bridging/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 """This module contains the support resources for the asset_bridging protocol.
 
-It was created with protocol buffer compiler version `libprotoc 27.2` and aea protocol generator version `1.0.0`.
+It was created with protocol buffer compiler version `libprotoc 31.1` and aea protocol generator version `1.0.0`.
```

Comment:
hmm interesting. I guess I updated my system and that takes precedence, or did you bump protoc?